### ISSUE TITLE
Different profiles

### ISSUE THREAD

**Issue by AndreaCrotti** [amacneil/dbmate#104]
Suppose I have some base database structure, which I can apply different changes to, what would be a good way to do that?

One way I thought I could do that could do something like:

- db/migrations/base.sql
- db/migrations/first/migration.sql
- db/migrations/first/base.sql -> ../base.sql
- db/migrations/second/migrations.sql
- db/migrations/second/base.sql -> ../base.sql

So using as symlink, and then call dbmate passing one or the other directory.
I think this would work, is there another better way to do something similar?

**Comment by amacneil** [comment]
You could use symlinks. That's probably the easiest way to do it, since we
don't have any concept of "imports" etc.

Can I ask why you want to do this? Dbmate usually assumes that your
application only has one schema.

**Comment by AndreaCrotti** [comment]
So to clarify the use case it's similar to just having two git branches, since one path is kind of experimental.
However it would be nice to have these two schemas in parallel, and we could anyway write the two schemas out to different databases locally.
So I think even that symlinks approach would allow that. It probably makes sense for dbmate to not support this, it's probably a very uncommon requirement.

**Comment by amacneil** [comment]
Agreed - I don't think this is really a use case that dbmate is designed to support. We assume that you have one canonical set of migrations, and that if you're running experimental branches they should live in git branches. For your described setup the symlinks should work fine.
